src/app/productConfig/productConfigForm/productConfigForm.component.spec.js

```diff
@@ -371,8 +371,14 @@ describe('product config form component', function () {
     it('orders frequency by name', () => {
       expect($ctrl.frequencyOrder({ name: 'NA' })).toEqual(0)
       expect($ctrl.frequencyOrder({ name: 'MON' })).toEqual(1)
-      expect($ctrl.frequencyOrder({ name: 'QUARTERLY' })).toEqual(2)
-      expect($ctrl.frequencyOrder({ name: 'ANNUAL' })).toEqual(3)
+      if (!$ctrl.hideQuarterly) {
```

Comment:
We shouldn't have conditionals in the test. We should have additional test cases that cover each of those possible combinations of `hideAnnual` and `hideQuarterly`.